litellm/integrations/opentelemetry.py

```diff
@@ -472,34 +510,6 @@ def set_attributes(self, span: Span, kwargs, response_obj):
                                     tool_calls[0].get("function").get("arguments"),
                                 )
 
-                # The unique identifier for the completion.
```

Comment:
moved this higher up, kept one code section just for message / response logging. Hoping to move this logic to our standard logging payload this week